packages/common/components/blocks/leadership-company-videos.marko

```diff
@@ -0,0 +1,34 @@
+import { get, getAsArray } from "@base-cms/object-path";
+import queryFragment from "../../graphql/fragments/leadership-company-videos";
+
+$ const { contentId } = input;
+
+<marko-web-query|{ nodes }|
+  name="related-published-content"
+  params={ contentId, limit: 20, queryFragment, includeContentTypes: ["Company"] }
+>
+  <if(nodes.length)>
+    <for|company| of=nodes>
+      <if(company.isLeader)>
+        $ const videos = getAsArray(company, "videos.edges").map(({ node }) => node);
+        <if(videos.length)>
+          <marko-web-node-list class="mt-block" collapsible=false>
+            <@header>
+              Videos from ${company.name}
+              <a target="_blank" href=get(company, "youtube.url") rel="noopener" class="small float-right py-1">
```

Comment:
Use the built-in `marko-web-link` component. This will handle the `rel` value when using targets